### ISSUE TITLE
UNDO feature no longer available after cache deletion

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#14823]
### Describe your problem!

A user reports on support mail (ans I can reproduce), that he is no longer able to perform an undo operation after accidentally deleting a cache.
The "undo toast" (or however that feature is called) is no longer shown.

### How to reproduce?

- Open a cache
- Tap on the trash can icon

### Actual result after these steps?

Cache is deleted
No undo button is shown

### Expected result after these steps?

Undo button should be shown

### Reproducible

Yes

### c:geo Version

2023.11.01

### Additional Information

Support ticket 396745

**Comment by fm-sys** [comment]
AFAIK the undo feature wasn't ever shown inside cache details but only in the cache list.

**Comment by SammysHP** [comment]
Correct. You can always tap on "store" again. But thinking about it again, that might be from the times when a cache could be only on one list. Pressing the "delete" button removes it from all lists, storing again doesn't restore it on all previous lists, right?

**Comment by Lineflyer** [comment]
I will ask the user, what his usecase was (respectively if he described it wrong).

**Comment by Lineflyer** [comment]
Here is a more specific description of the problem (in german for now):

Ich starte c:geo, 
öffne den Verlauf über das Verlauf-Symbol im Balken OBEN auf dem Startbildschirms, 
tippe dann auf das drei-Punkt-Menü,
auf "Caches verwalten",
dort auf "Alle vom Gerät löschen".

Es werden dann die Caches des Verlaufs gelöscht und ein Rückgängig machen ist nicht möglich.
Früher erschien an dieser Stelle für kurze Zeit eine Einblendung wie: "3 Caches gelöscht  Rückgängig". Die erscheint nun nicht.

Das Scheint aber anders zu sein, wenn ich den Verlauf über das Gespeichert-Symbol unten in der Schnellstart-Leiste öffne. Hier wurde mir gerade nach dem Löschen die entsprechende Einblendung gezeigt.
